ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nationalarchives/tdr-generated-graphql",
-  "version": "1.0.28",
+  "version": "1.0.29",
```

Comment:
Isn't this handled by the Jenkins build. Generates a new PR with version updated

Reply:
Same as the other ones, I needed them deployed to get the others to run.